plane/source/docs/guide-tilt-rotor.rst

```diff
@@ -0,0 +1,231 @@
+.. _guide-tilt-rotor:
+
+=================
+Tilt Rotor Planes
+=================
+
+Tilt rotors are treated by ArduPilot as a special type of
+QuadPlane. You should start off by reading the :ref:`QuadPlane
+documentation <quadplane-support` before moving onto this tilt-rotor
+specific documentation.
+
+In ArduPilot nomenclature, a tilt-rotor is a type of VTOL aircraft
+where transition between hover and forward flight is accomplished by
+tilting one or more rotors so that it provides forward thrust instead
+of upward thrust.
+
+This is distinct from :ref:`tailsitters <guide-tailsitter` where the
+autopilot and main fuselage change orientation when moving between
+hover and forward flight.
+
+Types of Tilt-Rotors
+====================
+
+ArduPilot supports a very wide range of tilt-rotor
+configurations. Common configurations include:
+
+- tilt-quadplanes with the front two motors tilting
+- tilt-quadplanes with all four motors tilting
+- tilt-tricopters with the front two motors tilting and rear tilt for
```

Comment:
tilt-tricopters with the front two motors tilting <for forward thrust> and rear tilt for
 +  yaw